src/tests/ftest/daos_test/rebuild.py

```diff
@@ -269,3 +269,18 @@ def test_rebuild_29(self):
         :avocado: tags=daos_test,daos_core_test_rebuild,test_rebuild_29
         """
         self.run_subtest()
+    def test_rebuild_30(self):
```

Comment:
pylint - there should be a blank line before the function defintion

Reply:
sure, thanks